docroot/profiles/custom/cgov_site/modules/custom/cgov_site_section/src/Plugin/Validation/Constraint/SectionParentTermConstraintValidator.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+namespace Drupal\cgov_site_section\Plugin\Validation\Constraint;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\Language\LanguageManagerInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+
+/**
+ * Validates the Section ParentTerm Constraint.
+ *
+ * @package Drupal\cgov_core\Plugin\Validation\Constraint
+ */
+class SectionParentTermConstraintValidator extends ConstraintValidator implements ContainerInjectionInterface {
+
+  /**
+   * Validator 2.5 and upwards compatible execution context.
+   *
+   * @var \Symfony\Component\Validator\Context\ExecutionContextInterface
+   */
+
+  protected $context;
+
+  /**
+   * The term storage.
+   *
+   * @var \Drupal\taxonomy\TermStorageInterface
+   */
+  protected $termStorage;
+
+  /**
+   * The language manager service.
+   *
+   * @var \Drupal\Core\Language\LanguageManagerInterface
+   */
+  protected $languageManager;
+
+  /**
+   * The entity type manager.
```

Comment:
This is the constructor for a validator, not an entity type manager. :)

Reply:
Updated the comment.